libdnf-cli/output/package_list_sections.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "libdnf-cli/output/package_list_sections.hpp"
+
+#include "libdnf-cli/tty.hpp"
+
+#include "libdnf/rpm/nevra.hpp"
+#include "libdnf/rpm/package_set.hpp"
+
+#include <libsmartcols/libsmartcols.h>
+
+#include <algorithm>
+#include <iostream>
+#include <string>
+
+namespace libdnf::cli::output {
+
+PackageListSections::PackageListSections() {
+    table = scols_new_table();
+    scols_table_enable_noheadings(table, 1);
+    if (libdnf::cli::tty::is_interactive()) {
+        scols_table_enable_colors(table, 1);
+        scols_table_enable_maxout(table, 1);
```

Comment:
In my opinion this setting makes the outputs (the info commands especially) less readable.

But mainly I would prefer if we chose one approach and used it through out dnf5.
Currently I see it is used only in repolist, transaction_table and here.

Reply:
I do not have any strong opinion on this. But I agree with the consistency so I've removed this setting.

Reply:
@m-blaha It seems there is now a missing new line at the end of all outputs.

I am not sure how it works but maybe since the option
```c++
scols_table_enable_maxout(table, 1);
```
was filling the window to the end it seemed as the prompt wrapped to a new line? Or perhaps it added a new line automatically.

Reply:
Oh, I'll check it. Thanks for catching it.

Reply:
I've looked how we deal with command output checking in our CI suite. The thing is, that we ignore any trailing new lines. I guess the reason is to keep the expected outputs in `Then stdout is` step cleaner - without the trailing empty line.
I did some preliminary testing with more strict checks and already discover a couple of issues (like redundant new lines in the end of output). But I'd like to fix this in a separate PR.
If you agree, I'll only fix the missing new line here and file an issue to make our output checks more strict regarding to trailing new lines.

Reply:
That sound good, thank you!